### PR TITLE
Debundle pycodestyle

### PR DESCRIPTION
**Pull request by pointlessone**
We have pep8 (renamed pycodestyle upstream) bundled and out code interwoven. This makes it hard to update pycodestyle. This PR removes bundled pep8 (2.2.1a0) and instead install upstream pycodestyle (2.3.1) as a dependency.

Additionally a bit of clean up was done that brings image size down to 59.2MB from the `latest` being 681MB (11.5x smaller).

### CODE REVIEW

**Comment by ABaldwinHunter** [comment]
@pointlessone nice work here! are there any breaking changes between the version of pep8 we were using and latest pycodestyle that would impact end users (e.g. with respect to configuration options)?

If so, maybe we will want to consider that in how we roll out update

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+engines:
+  pep8:
+    enabled: true
+ratings:
+  paths:
+  - "**.py"
```
Comment (ABaldwinHunter) [inline .codeclimate.yml:7]: We don't need `ratings: paths:` now in quality model world. I think this repo is still on Platform though, so I'll go ahead and migrate it. [edit: done!]

Also, wdyt about omitting exclude_paths if there are none

**Comment by pointlessone** [comment]
@ABaldwinHunter I don't think so but let me check and get back to you (tomorrow).

**Comment by pointlessone** [comment]
@ABaldwinHunter There seem to be no breaking changes between these releases in the [changelog](https://github.com/PyCQA/pycodestyle/blob/master/CHANGES.txt).

**Review by ABaldwinHunter — APPROVED** [review]
LGTM

**Pull request merged**